src/github/githubRepository.ts

```diff
@@ -75,6 +77,7 @@ export class GitHubRepository implements IGitHubRepository {
 		this._initialized = true;
 		if (!await this._credentialStore.hasOctokit(this.remote)) {
 			this._octokit = await this._credentialStore.login(this.remote);

```

Comment:
could we also log an event for auth failures, or something at the beginning of the login process, so that we get a sense of the auth success rate?